### PR TITLE
choco-pack: add page

### PR DESCRIPTION
**Pull request by pxgamer**
Part of #2016

----

## Basic Check

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

## References

Main page: https://chocolatey.org
GitHub repository: https://github.com/chocolatey/choco

### CODE REVIEW

File: pages/windows/choco-pack.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# choco pack
+
+> Package a NuGet specification into a nupkg file.
+
+- Package a NuGet specification to a nupkg file:
+
+`choco pack {{path/to/specification}}`
+
+- Package a NuGet specification using a specific version:
```
Comment (sbrl) [inline pages/windows/choco-pack.md]: Does this mean the version of the package that's being packed?

Comment (pxgamer) [reply pages/windows/choco-pack.md]: Yes, this is the version of the package that's being packed. 👍

Comment (sbrl) [reply pages/windows/choco-pack.md]: Ah, ok. Hrm. I wonder if there's a may to make it clearer?

What about `specifying the version of the resulting file`?

Comment (pxgamer) [reply pages/windows/choco-pack.md]: I guess that makes it clearer. :+1:

**Pull request merged**